src/app.js

```diff
@@ -0,0 +1,65 @@
+import {FilterableTable} from './components/FilterableTable.react';
+
+import * as SharedUtils from './utils/SharedUtils';
+
+document.addEventListener('DOMContentLoaded', function(){
+  var searchObject = SharedUtils.parseUrlSearchString(),
```

Comment:
Definitely. Do you think maybe parsing the DOM, parsing the query string and rendering is a good start for methods?